pkg/globalconfig/global_config.go

```diff
@@ -208,9 +224,9 @@ func ReadGlobalConfig() error {
 	if nodeps.NoBindMountsDefault == true {
 		DdevGlobalConfig.NoBindMounts = true
 	}
-	// For testing only, override UseTraefikDefault no matter what it's set to
-	if nodeps.UseTraefikDefault == true {
-		DdevGlobalConfig.UseTraefik = true
+	// For testing only, override UseTraditionalRouter no matter what it's set to
+	if nodeps.UseTraditionalRouter == true {
```

Comment:
```suggestion
	if nodeps.UseTraditionalRouter {
```

Never compare a bool value. Linter should anyway comlain here or we miss a config.